activation/activation.go

```diff
@@ -463,10 +462,10 @@ func (b *Builder) BuildNIPostChallenge(ctx context.Context, nodeID types.NodeID)
 		return challenge, nil
 	}
 
-	prev, err := b.cdb.GetLastAtx(nodeID)
+	prevAtx, err := b.GetPrevAtx(nodeID)
```

Comment:
As this `CachedDB.GetLastAtx` isn't used in other places, perhaps it is worth removing altogether?

Reply:
Removed it and a few others unused.